includes/Elements/Section/Section.php

```diff
@@ -515,6 +533,33 @@ public function before_render( $options ) {
 	 * @return void
 	 */
 	public function render( $options ) {
+		$masks = $options->get_value( 'shapes' );
+
+		if ( ! empty( $masks ) ) {
+			foreach ( $masks as $key => $shape ) {
```

Comment:
can you rename $shape to $shape_config? It is easier to read and understand it's purpose. Also, rename $key to $position so you can understand it's purpose when reading the code